terraform/cert-manager.tf

```diff
@@ -2,17 +2,19 @@ resource "kubectl_manifest" "application_argocd_cert_manager" {
   yaml_body = templatefile("${path.module}/templates/argocd-apps/cert-manager.yaml", {
     REPO_URL = local.repo_url
   })
+}
 
+resource "null_resource" "wait_for_cert_manager" {
```

Comment:
null_resource is deprecated. let's use what's suggested in the doc here: https://registry.terraform.io/providers/hashicorp/null/latest/docs/resources/resource